### PR TITLE
New version: PrismLauncher.PrismLauncher version 7.0

### PR DESCRIPTION
**Pull request by PrismAutomata**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109239&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/p/PrismLauncher/PrismLauncher/7.0/PrismLauncher.PrismLauncher.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+PackageVersion: '7.0'
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```
Comment (stephengillie) [inline manifests/p/PrismLauncher/PrismLauncher/7.0/PrismLauncher.PrismLauncher.installer.yaml:14]:
```suggestion
```

**Comment by stephengillie** [comment]
This package appears to also require a `JRE`: 
image

**Comment by Scrumplex** [comment]
> This package appears to also require a `JRE`: image

While that's true. Users are free to choose any JRE so we don't really want to force them to install some specific JRE using winget.

File: manifests/p/PrismLauncher/PrismLauncher/7.0/PrismLauncher.PrismLauncher.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ReleaseDate: 2023-06-08
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/PrismLauncher/PrismLauncher/releases/download/7.0/PrismLauncher-Windows-MSVC-Legacy-Setup-7.0.exe
+  InstallerSha256: 2C012ECF45B89E0D125A8317C6251ECEB12865B6D0C640BA5A2772EC71C7E3B7
+  Dependencies:
+    PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
+- Architecture: x64
+  InstallerUrl: https://github.com/PrismLauncher/PrismLauncher/releases/download/7.0/PrismLauncher-Windows-MSVC-arm64-Setup-7.0.exe
```
Comment (GurjinderPartola) [inline manifests/p/PrismLauncher/PrismLauncher/7.0/PrismLauncher.PrismLauncher.installer.yaml:21]: The download link for x64 is wrong, shouldn't it be `https://github.com/PrismLauncher/PrismLauncher/releases/download/7.0/PrismLauncher-Windows-MSVC-Setup-7.0.exe`?

Comment (Scrumplex) [reply manifests/p/PrismLauncher/PrismLauncher/7.0/PrismLauncher.PrismLauncher.installer.yaml:21]: That is indeed wrong. I guess WinGet Releaser failed to properly detect these executables. Let me fix this

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**